ui/src/app/components/help-sidebar.tsx

```diff
@@ -363,6 +381,18 @@ export const HelpSidebar = fp.flow(withCurrentWorkspace(), withUserProfile())(
       }
     }
 
+    renderCriteria() {
+      const {criteria} = this.props;
+      return <PopupTrigger
```

Comment:
I think the selections are meant to slide out, like how the help tips slide out when the info icon is clicked. We may want to verify before changing, but I believe that's what was discussed.

Reply:
Ok, wasn't aware of this. I'll change it

Reply:
Ok, let me know if you need to discuss, but should look similar to [this block](https://github.com/all-of-us/workbench/pull/3788/files#diff-52586f24a61602acc0422d94891b3666R530-R532) in the template and then add a new object for the icon to [icons array](https://github.com/all-of-us/workbench/pull/3788/files#diff-52586f24a61602acc0422d94891b3666L173). The hiding and showing of the content should already be built in.

Reply:
As discussed offline , i'll take care of this in another PR